hack/make/.integration-test-helpers

```diff
@@ -96,8 +97,16 @@ run_test_integration_suites() {
 				--junitfile="${ABS_DEST}/${pkgname//./-}-junit-report.xml" \
 				--raw-command \
 				-- go tool test2json -p "${pkgname}" -t ./test.main ${pkgtestflags}
-		); then exit 1; fi
+		); then
+			if [ -z "${TEST_INTEGRATION_NO_FAIL_FAST}" ]; then
```

Comment:
Should we reverse the condition? Don't fail fast by default, but allow opt-in to fail fast?

Reply:
That's what it's doing no? Also I'm not sure when someone would want to fail fast

Reply:
It's fail fast by default, to keep the previous behavior, but I'm fine with reversing it.

Reply:
Possibly, although in most cases where I'd run test-integration, I do so with a `TEST_FILTER=...` because I wouldn't even think of running all integration tests in my "inner loop" 🙈

Reply:
Hmmm, right, that's what the CI is for.. ;)

Reply:
Changed into `TEST_INTEGRATION_FAIL_FAST` which is an opt-in into old behavior.